src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.Configuration/ConfigurationClient.cs

```diff
@@ -4,44 +4,48 @@
 
 using Azure.Base.Diagnostics;
 using Azure.Base.Http;
+using Azure.Base.Http.Pipeline;
 using System;
 using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
 
-// TODO (pri 1): Add all functionality from the spec: https://msazure.visualstudio.com/Azure%20AppConfig/Azure%20AppConfig%20Team/_git/AppConfigService?path=%2Fdocs%2Fprotocol&version=GBdev
-// TODO (pri 1): Support "List subset of keys" 
-// TODO (pri 1): Support "Time-Based Access" 
-// TODO (pri 1): Support "KeyValue Revisions"
-// TODO (pri 1): Support "Real-time Consistency"
-// TODO (pri 2): Add support for filters (fields, label, etc.)
-// TODO (pri 2): Make sure the whole object gets deserialized/serialized.
-// TODO (pri 3): Add retry policy with automatic throttling
 namespace Azure.ApplicationModel.Configuration
 {
     public partial class ConfigurationClient
     {
-        const string SdkName = "Azure.Configuration";
-        const string SdkVersion = "1.0.0";
+        const string ComponentName = "Azure.Configuration";
+        const string ComponentVersion = "1.0.0";
+
+        static readonly HttpPipelinePolicy s_defaultRetryPolicy = RetryPolicy.CreateFixed(3, TimeSpan.Zero,
```

Comment:
You should also retry on 503 and 429 - at a minimum if there is a retry-after or x-ms-retry-after header.

Reply:
Thanks! I added 503. As to 429, we have a separate workitem to add it, as adding it requires inspection of the header and implementing throttling.